src/routes/app/residents/+page.server.ts

```diff
@@ -0,0 +1,70 @@
+import { PhoneRegex } from '$lib/regex/phone.js';
+import { logger } from '$lib/server/logger/index.js';
+import { prisma } from '$lib/server/prisma';
+import { fail } from '@sveltejs/kit';
+import zod from 'zod';
+
+export const load = async ({ locals }) => {
+	const residents = await prisma.resident.findMany({
+		where: {
+			tenantId: locals.tenant.id,
+		},
+		select: {
+			id: true,
+			name: true,
+			phoneNumber: true,
+		},
+	});
+
+	return {
+		residents: residents,
+	};
+};
+
+export const actions = {
+	default: async (event) => {
+		const form = await event.request.formData();
+
+		if (!form.has('name')) {
+			return fail(400, { error: 'phone_missing' });
```

Comment:
Errors could be an enum, to be used by the frontend

Reply:
I'd like to work out a system with you guys for handling these